saleor/graphql/schema.graphql

```diff
@@ -2662,6 +2645,28 @@ input OrderFilterInput {
   search: String
 }
 
+type OrderFulfill {
+  errors: [Error!]! @deprecated(reason: "Use typed errors with error codes. This field will be removed after 2020-07-31.")
+  fulfillments: [Fulfillment]
+  order: Order
+  orderErrors: [OrderError!]!
+}
+
+input OrderFulfillInput {
+  lines: [OrderFulfillLineInput!]!
+  notifyCustomer: Boolean
+}
+
+input OrderFulfillLineInput {
+  orderLineId: ID
+  stocks: [OrderFulfillStockInput!]!
+}
+
+input OrderFulfillStockInput {
```

Comment:
There's a `StockInput` type identical to this one, why create new one?

Reply:
Because they have different quantity descriptions.  Quantity in `StockInput` is a quantity that you want to have in stock. A `quantity` in `OrderFulfillStockInput` is a number of items to be fulfilled form the warehouse.

Reply:
Occam's razor vibes here